plugins/oss/terraformer/terraformer-local.yml

```diff
@@ -0,0 +1,15 @@
+profiles:
```

Comment:
Is this for example purposes or do we actually need this to start the service?

Reply:
I can exclude it from the terraformer's `kustomization.yml` file and mention this in the README if that's ok with you